configuration/config-txt/video.md

```diff
@@ -472,49 +445,96 @@ hdmi_mode=87
 hdmi_drive=2
 ```
 
-This may not work if your monitor does not support standard CVT timings.
 
-## LCD display/touchscreen options
+## Composite video mode options
+
+### sdtv_mode
+
+The `sdtv_mode` command defines the TV standard used for composite video output.
+
+| sdtv_mode | result |
+| --- | --- |
+| 0 | Normal NTSC |
+| 1 | Japanese version of NTSC – no pedestal |
+| 2 | Normal PAL |
+| 3 | Brazilian version of PAL – 525/60 rather than 625/50, different subcarrier |
+| 16 | Progressive scan NTSC |
+| 18 | Progressive scan PAL |
+
+### sdtv_aspect
+
+The `sdtv_aspect` command defines the aspect ratio for composite video output. The default value is `1`.
+
+| sdtv_aspect | result |
+| --- | --- |
+| 1 | 4:3 |
+| 2 | 14:9 |
+| 3 | 16:9 |
+
+### sdtv_disable_colourburst
+
+Setting `sdtv_disable_colourburst` to `1` disables colourburst on composite video output. The picture will be displayed in monochrome, but it may appear sharper.
+
+### enable_tvout (Pi 4 only)
+
+On the Raspberry Pi 4, composite output is disabled by default, due to the way the internal clocks are interrelated and allocated. Because composite video requires a very specific clock, setting that clock to the required speed on the Pi 4 means that other clocks connected to it are detrimentally affected, which slightly slows down the entire system. Since composite video is a less commonly used function, we decided to disable it by default to prevent this system slowdown. 
+
+To enable composite output, use the `enable_tvout=1` option. As described above, this will detrimentally affect performance to a small degree.
+
+
+## DSI display connector options
+
+These parameters allow configuration of displays connected to the DSI connected, such as the Raspberry Pi Touch Display.
```

Comment:
"displays connected to the DSI connected" ??

Reply:
Fixed